core/src/main/scala/sttp/tapir/TapirAuth.scala

```diff
@@ -34,8 +35,14 @@ object TapirAuth {
       challenge: WWWAuthenticateChallenge
   ): EndpointInput.Auth[T, EndpointInput.AuthType.Http] = {
     val codec = implicitly[Codec[List[String], T, CodecFormat.TextPlain]]
-    val authCodec =
-      Codec.list(Codec.string.map(stringPrefixWithSpace(authScheme))).mapDecode(codec.decode)(codec.encode).schema(codec.schema)
+    val authCodec = Codec
```

Comment:
I think a more appropriate comment for `bearer` and `basic` would now be sth like:

```
Reads authorization data from the `Authorization` headers starting with `Bearer `, removing the prefix.
```

Reply:
done